_posts/2024-12-19-moving-data-between-JBOD-disks-using-cruise-control.md

```diff
@@ -0,0 +1,345 @@
+---
+layout: post
+title: "Moving data between the JBOD disks using Cruise Control"
+date: 2024-12-19
+author: shubham_rawat
+---
+
+Apache Kafka is a platform which provides durability and fault tolerance by storing messages on disks and JBOD storage is one of the storage configuration types supported by Kafka.
```

Comment:
Kafka supports multiple disks. JBOD storage configuration type is more of a Strimzi thing. That is how we call it in the API. I would either reword it or change it from Kafka to Strimzi?